app/javascript/components/community/video-grid/index.tsx

```diff
@@ -1,13 +1,25 @@
-import { CommunityVideoModal } from '@/components/track/approaches-elements/community-videos/CommunityVideoModal'
-import { CommunityVideoAuthor } from '@/components/track/approaches-elements/community-videos/types'
-import React, { useState } from 'react'
+import React, { useCallback, useRef, useState } from 'react'
 import { QueryStatus } from 'react-query'
-import { Avatar, GraphicalIcon, Pagination } from '../../common'
+import { Request } from '@/hooks'
+import { ResultsZone } from '@/components/ResultsZone'
+import {
+  Avatar,
+  FilterFallback,
+  GraphicalIcon,
+  Pagination,
+} from '@/components/common'
+import type { VideoTrack } from '@/components/types'
+import { CommunityVideoModal } from '@/components/track/approaches-elements/community-videos/CommunityVideoModal'
 import { TrackFilterList } from './TrackFilterList'
-import { useVideoGrid } from './useVideoGrid'
+import { HandleTrackChangeType, useVideoGrid } from './useVideoGrid'
+import type { CommunityVideoType } from '@/components/types'
 
 type VideoGridProps = {
-  data: any
+  data: {
+    tracks: VideoTrack[]
+    request: Request
+    selectedTrackSlug: string | null
```

Comment:
I can't find any place where this field is used

Reply:
ok, I will remove this!